USER_GUIDE.md

```diff
@@ -0,0 +1,86 @@
+# User guide
+
+---
+Primitive guide for users 1.3.0 and above (need to expand)
+
+# App Engine configuration file deployment
+
+You can now deploy the cron/doc/etc. configuration files separately using the new goals:
+
+* `deployCron`
+* `deployDispatch`
+* `deployDos`
+* `deployIndex`
+* `deployQueue`
+
+## Source directory
+_For GAE Flexible projects_ The deployment source directory can be overridden by setting the `appEngineDirectory` parameter in the deploy configuration.
```

Comment:
I want to somehow emphasize, that it's only for flex projects. Maybe a different formatting/wording would be better? I meant it as a separate label, not part of the sentence starting with "The deployment..."